### ISSUE TITLE
viewDqDashboard not working in R 4.4.1 - Error in .make_numeric_version

### ISSUE THREAD

**Issue by joaofonsecapromptly** [OHDSI/DataQualityDashboard#555]
Using R 4.4.1, an error occurs when calling viewDqDashboard

```
Error in .make_numeric_version(x, strict, .standard_regexps()$valid_numeric_version) : 
  invalid non-character version specification 'x' (type: double)
```

Full example:

```
R version 4.4.1 (2024-06-14) -- "Race for Your Life"
Copyright (C) 2024 The R Foundation for Statistical Computing
Platform: aarch64-apple-darwin20

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> DataQualityDashboard::viewDqDashboard('/Users/joaofonseca/git/harmonizer/data_quality/output/dqd_results.json')
Error in .make_numeric_version(x, strict, .standard_regexps()$valid_numeric_version) : 
  invalid non-character version specification 'x' (type: double)

```

**Comment by katy-sadowski** [comment]
This will be fixed in the next release which is coming shortly (https://github.com/OHDSI/DataQualityDashboard/pull/547/files#diff-d01dbc4cce6a847d101ce48e580754f42c7a5bad466f7fbdcc9a2a226a512cc1)

**Comment by iamalonso** [comment]
Hi @katy-sadowski do you have any insights on this DataQualityDashboard issue?  I'm on R 4.4.1 and can't seem to run viewDqDashboard() as well 😔

**Comment by katy-sadowski** [comment]
I am hoping to have the new version including this bugfix released by the end of the week!

**Comment by katy-sadowski** [comment]
This should be fixed in the new v2.6.1 release. Please try it out and let me know if it works!

**Comment by iamalonso** [comment]
Hi @katy-sadowski, thank you! The error dissapeared.
